.github/workflows/build-workflow.yaml

```diff
@@ -37,3 +33,11 @@ jobs:
         run: |
           cd charts
           PATH=$PATH:/tmp/bin helm upgrade --install --set image.tag=v$GITHUB_SHA_SHORT --set rbac.create=true --wait --timeout 60s  log-router ./log-router
+      - name: build-chart
+        working-directory: charts/log-router
```

Comment:
could you add this in the release-workflow instead of build-workflow?
The release-workflow will work just for tag who is what we want